### DISCUSSION TITLE
Could not find to close newly open tab.

### DISCUSSION THREAD

**Discussion by neerajkumarlad** [webdriverio/webdriverio#8504] [Q&A] [answered]
Hi 

I am trying to work with multiple tabs but I am not getting to close newly open tab. 

I tried `browser.closeWindow()` but it just close all open browsers. 

Need anyone help to close newly open tab.

**Comment by christian-bromann** [comment, marked as answer]
As the docs say:

> The Close Window command closes the current top-level browsing context.

If you open the tab but haven't switched to it than calling `closeWindow` will close the window/tab you are currently on. If you like to close the window tab that was opened by your application, then you have to switch to it using [`switchWindow`](https://webdriver.io/docs/api/browser/switchWindow/).
